app/src/main/java/ru/netology/nmedia/repository/PostRepositoryImpl.kt

```diff
@@ -32,9 +35,16 @@ class PostRepositoryImpl @Inject constructor(
     private val apiService: ApiService,
     private val auth: AppAuth,
 ) : PostRepository {
-    override val data = postDao.getAll()
+
+    private val pageSize = 10
```

Comment:
1. `pageSize` - константа. 
   1. Для константы в котлине есть ключевое слово `const`. 
   2. Константы традиционно именуются в `CAPS_CASE`.
   3. Если коротко, то:  
   ```suggestion
       private companion object {
           private const val PAGE_SIZE = 10
       }
   ```
2. `pageSize` используется только в одном месте:
   1. Вы уверены, что Вам нужно тогда отдельно выделять эту константу?
   2. Может есть смысл использовать её же в `PagingConfig`?

Reply:
Максим, большое спасибо за подробную обратную связь!